### ISSUE TITLE
ReaR 2.5, CentOS7.9, Rubrik CDM configuration not creating functional ISO. No specific error in debug log.

### ISSUE THREAD

**Issue by cervantevo** [rear/rear#2678]
#### Relax-and-Recover (ReaR) Issue Template

#Fill in the following items before submitting a new issue
(quick response is not guaranteed with free support):

- ReaR version ("/usr/sbin/rear -V"):
Relax-and-Recover 2.5 / 2019-05-10

OS version ("cat /etc/os-release" or "lsb_release -a" or "cat /etc/rear/os.conf"): 
OS_VENDOR=CentOS
OS_VERSION=7

- ReaR configuration files ("cat /etc/rear/site.conf" and/or "cat /etc/rear/local.conf"):
OUTPUT=ISO
BACKUP=CDM

REAL_MACHINE=x86_64
ARCH=Linux-x86_64
OS_MASTER_VENDOR="CentOS"
OS_MASTER_VERSION="7"
OS_MASTER_VENDOR_ARCH="CentOS/x86_64"
OS_MASTER_VENDOR_VERSION="CentOS/7"
OS_MASTER_VENDOR_VERSION_ARCH="CentOS/7/x86_64"
OS_VENDOR=CentOS
OS_VERSION=7.9.2009
OS_VENDOR_ARCH="CentOS/x86_64"
OS_VENDOR_VERSION="CentOS/7"
OS_VENDOR_VERSION_ARCH="CentOS/7/x86_64"

BOOTLOADER="GRUB2-EFI"
export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:/usr/lib64/bind9-export"
MODULES=( 'loaded_modules' )
USE_DHCLIENT=no
COPY_AS_IS_EXCLUDE=(/var/log/lastlog)
OUTPUT_URL=null
KEEP_OLD_OUTPUT_COPY=yes

- Hardware (PC or PowerNV BareMetal or ARM) or virtual machine (KVM guest or PoverVM LPAR): 
VMWare 6.7 

- System architecture (x86 compatible or PPC64/PPC64LE or what exact ARM device): 
x86_64

- Firmware (BIOS or UEFI or Open Firmware) and bootloader (GRUB or ELILO or Petitboot): 
UEFI GRUB2

- Storage (local disk or SSD) and/or SAN (FC or iSCSI or FCoE) and/or multipath (DM or NVMe): 
local 

- Storage layout ("lsblk -ipo NAME,KNAME,PKNAME,TRAN,TYPE,FSTYPE,SIZE,MOUNTPOINT" or "lsblk" as makeshift): 
NAME                     KNAME     PKNAME    TRAN   TYPE FSTYPE       SIZE MOUNTPOINT
/dev/sda                 /dev/sda                   disk               20G
|-/dev/sda1              /dev/sda1 /dev/sda         part vfat         200M /boot/efi
|-/dev/sda2              /dev/sda2 /dev/sda         part ext4           1G /boot
`-/dev/sda3              /dev/sda3 /dev/sda         part LVM2_member 18.8G
  |-/dev/mapper/vg0-root /dev/dm-0 /dev/sda3        lvm  ext4        16.8G /
  `-/dev/mapper/vg0-swap /dev/dm-1 /dev/sda3        lvm  swap           2G [SWAP]
/dev/sr0                 /dev/sr0            sata   rom              1024M

-Description of the issue (ideally so that others can reproduce it):

Attempting to setup a working ReaR (2.5), CentOS (7.9), and Rubrik CDM (5.3) backup/restore process, testing with this setup currently is not working though. 
I have had to specify all of the vendor and arch version as the correct info didn't seem to be gathered appropriately. 
That assumption was based on the log output prior to specifying these things, it would create an ISO using Fedora and i386 arch references. 

In either instance, the output log states the process failed, but the STDOUT doesn't state any failure. Looking in the logs, I don't see the definative ERROR to resolve. 
One process does create an ISO, but the recovery doesn't work (won't properly prompt for CDM recovery via ReaR), and the other way no ISO is created. 

Need to understand at what point either method is truly failing. 

-Workaround, if any:
none found

-Attachments, as applicable ("rear -D mkrescue/mkbackup/recover" debug log files):
Log below is using the full local.conf printed above.
[rear-mjgtest-03-l30.log](https://github.com/rear/rear/files/7165149/rear-mjgtest-03-l30.log)

Log below is when using minimal OS/Vendor descriptors options in the local.conf
[rear-limitedlocal.log](https://github.com/rear/rear/files/7165153/rear-limitedlocal.log)

To paste verbatim text like command output or file content,
include it between a leading and a closing line of three backticks like
````
```
verbatim content
```
````

**Comment by DamaniN** [comment]
ReaR v2.6 is required for CDM support.

**Comment by cervantevo** [comment]
Thank you. I will retry with that version. Rubrik documentation, or support did not point this out. I will note however, that I did try with that version initially and received similar if not the exact outcome I have with v.2.5 described here. But I did not keep the logs from that attempt (of course). So I will recreate this best I can and reopen if necessary.

**Comment by jsmeix** [comment]
@cervantevo FIY:
In general you may also have a look at the section
"Testing current ReaR upstream GitHub master code" in
https://en.opensuse.org/SDB:Disaster_Recovery
where I describe how one could test several ReaR versions in parallel
each one in its own separated directory without conflicts between each other
and without conflicts with a normally installed ReaR version via RPM package.
